library/std/src/collections/hash/map.rs

```diff
@@ -2783,13 +2782,24 @@ impl RandomState {
         // iteration order allows a form of DOS attack. To counter that we
         // increment one of the seeds on every RandomState creation, giving
         // every corresponding HashMap a different iteration order.
-        thread_local!(static KEYS: Cell<(u64, u64)> = {
-            Cell::new(sys::hashmap_random_keys())
+        thread_local!(static KEYS: Cell<[u64; 2]> = {
```

Comment:
It was copied from the previous PR. I think at the time I introduced this change because `[u64; 2]` is slightly more idiomatic than `(u64, u64)`.

Reply:
I personally think `(u64, u64)` makes more sense because it's used to fill out a `RandomState` which consists of two separate `u64`s rather than an array, but it's not a huge deal.